src/app.tsx

```diff
@@ -9,18 +9,12 @@ import Footer from "./components/footer";
 import Form from "./components/form";
 import Results from "./components/results";
 
-type AppState = {
-  data: object;
-  requestParams: any;
-};
-
-export default class App extends React.Component {
-  state: AppState = {
-    data: {},
-    requestParams: {},
-  };
+ 
+const App = () => {
+  const [data, setData] = useState({});
+  const [requestParams, setRequestParams] = useState({});
```

Comment:
In your useState({}) try adding `method: "get", url:"something"` between the curly brackets